From patchwork Sat Nov  7 21:49:41 2009
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [16/75] drx397xD: declare MODULE_FIRMWARE
Date: Sat, 07 Nov 2009 21:49:41 -0000
From: Ben Hutchings <ben@decadent.org.uk>
X-Patchwork-Id: 58400

Move punctuation into definition of _FW_ENTRY so we can use either a
comma or semicolon.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>

---
drivers/media/dvb/frontends/drx397xD.c    |    7 ++++---
 drivers/media/dvb/frontends/drx397xD_fw.h |    4 ++--
 2 files changed, 6 insertions(+), 5 deletions(-)

--- a/drivers/media/dvb/frontends/drx397xD.c
+++ b/drivers/media/dvb/frontends/drx397xD.c
@@ -39,7 +39,7 @@ static const char mod_name[] = "drx397xD
 #define F_SET_0D4h	2
 
 enum fw_ix {
-#define _FW_ENTRY(a, b, c)	b
+#define _FW_ENTRY(a, b, c)	b,
 #include "drx397xD_fw.h"
 };
 
@@ -77,7 +77,7 @@ static struct {
 			.file	= NULL,					\
 			.lock	= __RW_LOCK_UNLOCKED(fw[c].lock),	\
 			.refcnt = 0,					\
-			.data	= { }		}
+			.data	= { }		},
 #include "drx397xD_fw.h"
 };
 
@@ -1507,4 +1507,5 @@ EXPORT_SYMBOL(drx397xD_attach);
 MODULE_DESCRIPTION("Micronas DRX397xD DVB-T Frontend");
 MODULE_AUTHOR("Henk Vergonet");
 MODULE_LICENSE("GPL");
-
+#define _FW_ENTRY(a, b, c) MODULE_FIRMWARE(a);
+#include "drx397xD_fw.h"
--- a/drivers/media/dvb/frontends/drx397xD_fw.h
+++ b/drivers/media/dvb/frontends/drx397xD_fw.h
@@ -18,8 +18,8 @@
  */
 
 #ifdef _FW_ENTRY
-	_FW_ENTRY("drx397xD.A2.fw",	DRXD_FW_A2 = 0,	DRXD_FW_A2	),
-	_FW_ENTRY("drx397xD.B1.fw",	DRXD_FW_B1,	DRXD_FW_B1	),
+	_FW_ENTRY("drx397xD.A2.fw",	DRXD_FW_A2 = 0,	DRXD_FW_A2	)
+	_FW_ENTRY("drx397xD.B1.fw",	DRXD_FW_B1,	DRXD_FW_B1	)
 #undef _FW_ENTRY
 #endif /* _FW_ENTRY */
 
